### PR TITLE
feat: Added config.yml file for Welcome Bot

### PR DESCRIPTION
**Pull request by m-code12**
## Issue that this pull request solves

 Closes: #274 

### What is the change?
Created config.yml file for the welcome bot to help first-time contributors. To activate the app you need to go to https://github.com/apps/welcome and click on install and give the app the required permissions  (Pull requests: Read & Write
Issues: Read & Write).

### How was it tested?
I have tested the welcome bot with the config.yml file in my personal repository. Here is a link to the repo:
https://github.com/m-code12/WelcomeBot

## Submissions guide:

- [ ] Have you made corresponding changes to the documentation?
- [x] Your submission doesn't break any existing feature.
- [ ] Have you lint your code locally prior to submission?

### CODE REVIEW

File: .github/config.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Configuration for welcome - https://github.com/behaviorbot/welcome
+
+# Configuration for new-issue-welcome - https://github.com/behaviorbot/new-issue-welcome
+# Comment to be posted to on first time issues
+
+newIssueWelcomeComment: >
+  Hello there!👋 Welcome to the project!🚀⚡
+  
+  
+  Thank you and congrats🎉for opening your very first issue in this project.Canvasboard is an interactive board with plugins focusing on replacing the need of multiple platforms for teaching, presenting or explaining your ideas. The interactive plugins are built using web technologies to ease your work right within a single platform.
```
Comment (Kajol-Kumari) [inline .github/config.yml:10]: can you please provide a pace before `for opening ...`

File: .github/config.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  Please adhere to our [Code of Conduct](https://github.com/Canvasbird/canvasboard/blob/master/CODE_OF_CONDUCT.md).🙌 We will get back to you as soon as we can 😄.
+  
+  Feel free to join our [Slack Community](https://join.slack.com/t/canvasbird-workspace/shared_invite/zt-jjdml591-~SsL4vdSX9Mc_GUVIvYV7g).💖 We have different channels for active discussions.✨ Hope you have a great time there!😄
+  
+  
+# Configuration for first-pr-merge - https://github.com/behaviorbot/first-pr-merge
+# Comment to be posted to on pull requests merged by a first time user
+
+firstPRMergeComment: >
+  Congrats on merging your first pull request! 🎉 All the best for your amazing open source journey ahead 🚀.
```
Comment (Kajol-Kumari) [inline .github/config.yml:36]: please leave an extra blank line at the end

File: .github/config.yml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  
+  
+# Configuration for new-pr-welcome - https://github.com/behaviorbot/new-pr-welcome
+# Comment to be posted to on PRs from first time contributors in your repository
+
+newPRWelcomeComment: >
+  Hello there!👋 Welcome to the project!💖
+  
+  
+  Thank you and congrats🎉 for opening your first pull request ✨.Canvasboard is an interactive board with plugins focusing on replacing the need of multiple platforms for teaching, presenting or explaining your ideas. The interactive plugins are built using web technologies to ease your work right within a single platform.
```
Comment (Kajol-Kumari) [inline .github/config.yml:26]: here, please change ` request ✨.` to ` request.✨`

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
@m-code12 please incorporate these minor requested changes and it will be good to go :)

**Comment by m-code12** [comment]
@Kajol-Kumari I have made all the changes.

**Review by Kajol-Kumari — APPROVED** [review]
Thanks for incorporating the requested changes @m-code12 
LGTM :+1:

**Comment by Kajol-Kumari** [comment]
Updated your Score for NWoC. Please verify it once on your end :)

**Comment by m-code12** [comment]
I guess there is some issue with the Njack Leaderboard page, it shows data could not be parsed.
Also, you have added the points for opening the welcome bot issue #274 , right?

**Comment by Kajol-Kumari** [comment]
yeah, I have done +7 which includes for both PR an d the issue

**Pull request merged**